hw/ip/otbn/dv/model/iss_wrapper.h

```diff
@@ -44,7 +44,7 @@ struct ISSWrapper {
 
   // Provide data for RND. ISS will stall when RND is read and RND data isn't
   // available
-  void edn_rnd_data(uint32_t edn_rnd_data[8]);
+  void edn_step(uint32_t edn_rnd_data, int edn_rnd_data_valid);
```

Comment:
The naming here is a bit confusing. I think that `edn_rnd_data` is considered iff `edn_rnd_data_valid` is false! Maybe we should be renaming the second signal to something like `edn_rnd_cdc_complete`?